### ISSUE TITLE
Why didn't you normalize image data?

### ISSUE THREAD

**Issue by sgmath12** [wkentaro/pytorch-fcn#125]
In common setting, we normalize the data something like this  _(img - img.mean)/(img.std)_

But in your code, you didn't divide std.

Is there any reason for that?

**Comment by wkentaro** [comment]
Because vgg16 doesn't, which is used as a pretrained-model for feature extraction
